classifier.py

```diff
@@ -54,20 +54,27 @@ def __init__(self, config):
         self.pooler_dropout = nn.Dropout(config.hidden_dropout_prob)
         self.proj = nn.Linear(config.hidden_size, config.num_labels)
 
-    def forward(self, input_ids, attention_mask):
+    def embed(self, input_ids):
+        '''Takes a batch of sentences and returns BERT embeddings'''
+        return self.bert.embed(input_ids=input_ids)
+
+    def forward(self, input_ids, attention_mask, forward_embedding=False):
```

Comment:
change `input_ids` to `ids_or_embedding`

then change forward_embedding to `is_embedding`